src/opm/output/eclipse/Summary.cpp

```diff
@@ -459,6 +459,14 @@ double efac( const std::vector<std::pair<std::string,double>>& eff_factors, cons
     return (it != eff_factors.end()) ? it->second : 1;
 }
 
+inline quantity alqrate( const fn_args& args ) {
+    const auto& well = args.schedule_wells.front();
+    const auto& name = well.name();
+    if( args.wells.count( name ) == 0 ) return { 0, measure::gas_surface_rate };
+    const auto alq_value = args.wells.at(name).rates.get(rt::alq, 0.0);
```

Comment:
I just remembered that this will look up the well results twice.  If you rewrite it as below you'll only need to traverse the well results once.
```C++
auto xwPos = args.wells.find(well.name());
if (xwPos == args.wells.end()) {
    return { 0.0, measure::gas_surface_rate };
}

return { xwPos->second.rates.get(rt::alq, 0.0), measure::gas_surface_rate };
```